### ISSUE TITLE
Access item at index

### ISSUE THREAD

**Issue by thijstriemstra** [RobTillaart/SET#5]
I would like to access the 2nd, 3rd, etc item in a Set, but I couldn't find something like `at()` or `get()`.

Is this possible?

**Comment by RobTillaart** [comment]
Yes, something like 

```cpp
int getNth(uint_t n)
{
  if (n==1) return s.first();
  s.first();
  for (uint8_t i = 0; i < n-2; i++) s.next();
  return s.next();
}
```
(not tested)

It can be speed up a bit (few %) by rewriting it low level.

**Comment by thijstriemstra** [comment]
Thanks. Will this be added to the library?

**Comment by RobTillaart** [comment]
Not this afternoon, but yes I think it is a useful function.

Would start with the above "reference", but I see already 3 ways to improve performance.

it has a bug too as it does not handle 0 correctly ...

**Comment by RobTillaart** [comment]
@thijstriemstra 
Merged **getNth()** into 0.2.4
It is a sort of reference implementation, not fully optimized. 
Might be done later...

**Comment by thijstriemstra** [comment]
nice, thanks!
